logstash/tests/logstash/logstash.tests.bom

```diff
@@ -17,6 +17,7 @@ brooklyn.catalog:
                 input {
                   file {
                     path => "/qa-test/*.log"
+                    start_position => "beginning"
```

Comment:
According to the docs:
> Note that the rotated filename will be treated as a new file so if start_position is set to beginning the rotated file will be reprocessed.

Can this be an issue if we have different files on rotation? I guess we don't really care as this is for the tests but maybe the test expects only one generated log file?

Reply:
As you say this is only for the tests.  No log rotation is going on because we are just creating the input file with a bash loop so won't be an issue here.